src/simulator_core/solver/network/assets/base_item.py

```diff
@@ -48,6 +48,21 @@ def set_matrix_index(self, index: int) -> None:
         """
         self.matrix_index = index
 
+    def get_index_matrix(
+        self, property_name: str, connection_point: int, use_relative_indexing: bool
```

Comment:
Waarom zou je nog steeds matrix: bool = True gebruiken?

Reply:
Je roept hem niet standaard aan omdat je altijd get_index_matrix wilt krijgen, waardoor je alleen in het geval dat je het niet wilt de matrix flag set.

Reply:
offline besproken